### PR TITLE
[nodejs] Add appmetrics-dash as a dev dependency

### PR DESCRIPTION
**Pull request by seabaylea**
The Eclipse Codewind project uses the `appmetrics-dash` NPM module to provide development time application monitoring and performance testing data.

This updates the nodejs and nodejs-express stacks to add appmetrics-dash as a devDependency.

For the nodejs stack, this is done using `attach()`, which means it will only be added if the app provides some form of HTTP server. This is deliberate so that adding the dashboard doesn't prevent applications that are batch jobs etc from exiting.

### CODE REVIEW

**Comment by seabaylea** [comment]
@neeraj-laad Updates made.

File: incubator/nodejs/image/project/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "nodejs",
+  "version": "0.2.0",
```
Comment (neeraj-laad) [inline incubator/nodejs/image/project/package.json]: This should be 0.2.1 to keep it in line with the stack version

Comment (seabaylea) [reply incubator/nodejs/image/project/package.json]: Done

**Comment by seabaylea** [comment]
@neeraj-laad updates done

**Pull request merged**